docs/fides/docs/language/resources/registry.md

```diff
@@ -11,14 +11,13 @@ A Registry is a collection of System resources. You add a system to a Registry b
 * A System may belong to only one Registry.
 
 * All Registries are siblings: You cannot create a hierarchy of Registries.
-* Collecting your systems into Registries is optional. 
-
+* Collecting your systems into Registries is optional.
 
 ## Object Structure
 
-**fides_key**<span class="required"/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;_string_
+**fides_key**<span class="required"/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;_constrained string_
 
-A string token of your own invention that uniquely identifies this Registry. It's your responsibility to ensure that the value is unique across all of your Registry objects. The value may only contain alphanumeric characters and underbars (`[A-Za-z0-9_]`). 
+A string token of your own invention that uniquely identifies this Registry. It's your responsibility to ensure that the value is unique across all of your Registry objects. The value may only contain alphanumeric characters and underbars (`[A-Za-z0-9_.-]`).
```

Comment:
```suggestion
A custom string token that uniquely identifies this Registry. It must be unique across all Registry objects. The value may only contain alphanumeric characters, `_`, `.`, and `-` (it will be evaluated against the regular expression: `[A-z0-9_.-]`).
```